src/act.drive.cpp

```diff
@@ -1317,9 +1325,9 @@ ACMD(do_driveby)
   order_list(list);
   roll_individual_initiative(vict);
   if (GET_INIT_ROLL(vict) < 12)
-    send_to_char(vict, "You suddenly feel a cap busted in your ass as %s zooms past!\r\n", GET_VEH_NAME(ch->in_veh));
+    send_to_char(vict, "You suddenly feel force slamming into your hoop as %s zooms past!\r\n", GET_VEH_NAME(ch->in_veh));
   else
-    send_to_char(vict, "A hail of bullets flies from %s as it zooms past!\r\n", GET_VEH_NAME(ch->in_veh));
+    send_to_char(vict, "A flurry of attacks flies from %s as it zooms past!\r\n", GET_VEH_NAME(ch->in_veh));
```

Comment:
This doesn't read well even if it's purely melee attacks. Something needs to change, probably along the lines of being more specific about what attack is being done and with what melee weapon.